src/Admin/AbstractAdmin.php

```diff
@@ -664,10 +664,14 @@ public function getDataSourceIterator()
 
         foreach ($this->getExportFields() as $key => $field) {
             $label = $this->getTranslationLabel($field, 'export', 'label');
+
+            // NEXT_MAJOR: We have to find another way to have a translated label or stop deprecating the translator.
```

Comment:
Do we need this comment? IMHO creating a new issue with the problem would be enough. 

`NEXT_MAJOR` comments are simple instruction that need to be done on the main branch.

Reply:
Well, I agree and not agree at the same time. 

Agree on the `NEXT_MAJOR` comments should be simple instructions to follow.

But here we have already some instructions that could not be followed, so we either improve those intructions or we remove them because they cannot be followed (and we can create an issue about that, just in case someone wants to improve on the topic)